BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/ContentType/ContentType.js

```diff
@@ -15,7 +15,7 @@ class ContentType extends OrchestraModel
             case "read":
                 return Routing.generate('open_orchestra_api_content_type_show', urlParameter);
             case "delete":
-                urlParameter.contentTypeId = this.get('content_type_id');
+                urlParameter.contentTypeId = this.get('id');
```

Comment:
I would prefer to keep this.get('content_type_id') and change idAttribute in model.